test/sqllogictest/table_func.slt

```diff
@@ -314,3 +309,94 @@ SELECT abs(generate_series) FROM generate_series(-1, 2), repeat_row(generate_ser
 
 statement error  Negative multiplicity in constant result: -1
 SELECT * FROM (values ('a')), repeat_row(-1)
+
+query error unable to determine which implementation to use
+SELECT generate_series FROM generate_series(null, null, null)
+----
+
+query I
+SELECT generate_series FROM generate_series(1, 3, 1)
+----
+1
+2
+3
+
+query I
+SELECT generate_series FROM generate_series(1, 100, 0)
+----
+
+query I
+SELECT generate_series FROM generate_series(1, 10, 11)
+----
+1
+
+
+query I
+SELECT generate_series FROM generate_series(3, 1, -1)
+----
+1
+2
+3
+
+query I
+SELECT generate_series FROM generate_series(3::bigint, 1::bigint, -1::bigint)
```

Comment:
Not sure what the semantics of all of these tests are, but would you mind expanding the coverage of the `i64` version of `generate_series`?